ChangeLog.md

```diff
@@ -20,6 +20,12 @@ See docs/process.md for more on how version tagging works.
 
 Current Trunk
 -------------
+- The WebAssembly memory used by emscripten programs is now, by default, created
+  in the wasm file and exported to JavaScript.  Previously we could create the
+  memory in JavaScript and import it into the wasm file.  The new
+  `IMPORTED_MEMORY` setting can be used to revert to the old behaviour.  This
+  can be useful, for example, if the wasm memory needs to be created at runtime
+  before wasm instantiation.
```

Comment:
Maybe add a comment about the breaking changes here, like people providing the memory?

Reply:
Done.. plus I've included assertions that point ppl to this settings if they try to do that without this setting.